initial/src/main/java/com/example/services/CourseService.java

```diff
@@ -0,0 +1,22 @@
+package com.example.services;
+
+import com.example.springboot.Course;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+
+@RestController
+public class CourseService {
+    // go through static modifiers & public/private modifiers for java methods
```

Comment:
When you have an action item for yourself, make to add comments like below: 
```
// TODO: xxxxxx
```
And btw, please remember to go over the fundamentals you are mentioning here. :)

Reply:
updated my comment using TODO for the time being